src/code/z_camera.c

```diff
@@ -464,10 +458,10 @@ Vec3s* Camera_GetBgCamFuncData(Camera* camera) {
  */
 s32 Camera_GetBgCamIndex(Camera* camera, s32* bgId, CollisionPoly* poly) {
     s32 bgCamIndex;
-    PosRot playerPosRot;
+    UNUSED PosRot playerPosRot;
```

Comment:
We don't use unused in oot so far #1321

Reply:
It's worth noting that the are already 3 uses of `UNUSED` in z_camera.c, and that a lot of warnings are added without the `UNUSED`. But I'll leave it to that PR to sort out, so I've removed them